packages/core/package.json

```diff
@@ -62,19 +62,12 @@
   "devDependencies": {
     "@open-wc/karma-esm": "2.16.17",
     "@skypack/package-check": "0.2.2",
-    "@storybook/addon-a11y": "6.0.21",
-    "@storybook/addon-actions": "6.0.21",
-    "@storybook/addon-backgrounds": "6.0.21",
-    "@storybook/addon-controls": "6.0.21",
-    "@storybook/addon-cssresources": "6.0.21",
-    "@storybook/addon-docs": "6.0.21",
-    "@storybook/addon-knobs": "6.0.21",
-    "@storybook/addon-links": "6.0.21",
-    "@storybook/addon-storysource": "6.0.21",
-    "@storybook/addon-viewport": "6.0.21",
-    "@storybook/addons": "6.0.21",
-    "@storybook/cli": "6.0.21",
-    "@storybook/web-components": "6.0.21",
```

Comment:
The addon-essentials package storybook provides combines most of these into a single base package simplifying the number of dependencies needed.